src/app/components/content/IsaacQuestion.tsx

```diff
@@ -125,17 +109,15 @@ export const IsaacQuestion = withRouter(({doc, location}: {doc: ApiTypes.Questio
                 </div>}
 
                 {/* CS Hints */}
-                {isCS && <IsaacLinkHints questionPartId={doc.id as string} hints={doc.hints} />}
+               <IsaacLinkHints questionPartId={doc.id as string} hints={doc.hints} />
 
                 {/* Validation Response */}
                 {showQuestionFeedback && validationResponse && !canSubmit && <div className={`validation-response-panel p-3 mt-3 ${correct ? "correct" : ""}`}>
                     <div className="pb-1">
-                        <h1 className="m-0">{sigFigsError ? "Significant Figures" : correct ? "Correct!" : "Incorrect"}</h1>
+                        <h1 className="m-0">{correct ? "Correct!" : "Incorrect"}</h1>
                     </div>
                     {validationResponse.explanation && <div className="mb-2">
-                        {invalidFormatError ? invalidFormatFeeback : tooManySigFigsError ? tooManySigFigsFeedback : tooFewSigFigsError ? tooFewSigFigsFeedback :
```

Comment:
I'm glad this aberration is gone. These chained ternary operators are very hard to read and should be avoided.